visearch-android/src/main/java/com/visenze/visearch/android/model/FacetRange.java

```diff
@@ -4,24 +4,24 @@
 
 public class FacetRange {
     @SerializedName("min")
-    private Number min;
+    private String min;
```

Comment:
how come this is String? should be number

Reply:
this is changed by shannon. he tested string does not work.

Reply:
ok